models/articles-model.js

```diff
@@ -4,8 +4,11 @@ const db = require("../db/connection.js");
 //FUNCTION
 function selectArticle(article_id) {
    return db.query("SELECT * FROM articles WHERE article_id = $1", [article_id])
-      .then((article) => {
-        return { article: article.rows[0] };
+     .then(({ rows }) => {
+       if (rows.length === 0) {
+         next(err);
+       }
+        return rows;
```

Comment:
For single articles, we do need to access that first index at [0] as rows is an array